bsip-0061.md

```diff
@@ -0,0 +1,65 @@
+
+# Motivation
+At present, the BitShares protocol does not allow direct modification of limit orders in the markets. Instead, to modify a limit order, the order must be canceled and and a new one created in its place. This results in greater-than-necessary overhead when updating a limit order, an action which is performed by market maker robots with great frequency (hundreds of times per day, per market). Additionally, the cancel/create combination requires payment of two fees, which has led to some pressure from market makers for BitShares to provide a cheaper mechanism for maintaining fresh order walls in the markets.
+
+# Rationale
+The specification below was chosen on the following criteria:
+ - Minimal storage space for historical data (i.e. serialized operation size)
+   - Cancel/create combination requires two distinct operations, and explicit restatement of all order parameters
+   - Update operation requires only one operation, which elides all fields which are not being updated
+ - Minimal processing
+   - No deletion/construction is necessary, object is modified in place
+   - Cancel/create requires two separate modifications of balances; update requires zero or one
+   - Order matching is only triggered if order is moved past previous top of book
+ - Minimal RAM consumption for undo storage
+   - Object modification requires slightly less space than object deletion and creation
+   - Balances and account statistics are not updated unless necessary, avoiding unnecessary undo storage
+ - Finer granularity of fees
+   - Low fee if only the order object is modified
+   - Higher fee if balances must be updated as well
+
+# Specification
+A complete implementation of this BSIP is available for review [here](https://github.com/bitshares/bitshares-core/compare/develop...nathanhourt:issue-1604).
+
+The `limit_order_update_operation` contains the following fields:
+```cpp
+       asset fee;
+       account_id_type seller;
+       limit_order_id_type order;
+       optional<price> new_price;
+       optional<share_type> delta_amount_to_sell;
+       optional<time_point_sec> new_expiration;
+
+       extensions_type extensions;
+```
+
+A single new evaluator is required, which shall check that the new field values are valid for the order being updated, and update the order and balances as necessary, then trigger order matching only if the price was modified such that the order has moved past the previous top-of-book order.
+
+No new indexes are required. No modifications to existing indexes are required. No new database objects are defined. No modifications to existing database objects are required.
+
+# Discussion and Summary for Shareholders
```

Comment:
Consider changing this to just "Summary" and Replacing "See Also" with "Discussion"